### PR TITLE
Update authors on Readme.md

### LINKED ISSUE #1: Update authors on Readme.md

### PR DESCRIPTION
**Pull request by lucidbard**
Fixes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,9 @@
 
 Hello World, this is my second and final contribution!
 Hello class, this is Julian's contribution.
+
+Feature Branches
+
+AUTHORS:
+ - lucidbard
```
Comment (lucidbard) [inline README.md:9]: This is me

File: README.md [hunk 2]
```diff
@@ -2,3 +2,9 @@
 
 Hello World, this is my second and final contribution!
 Hello class, this is Julian's contribution.
+
+Feature Branches
```
Comment (lucidbard) [inline README.md:6]: You should include correct format here.

File: README.md [hunk 3]
```diff
@@ -2,3 +2,9 @@
...
+
+AUTHORS:
```
Comment (jbonilla2K) [inline README.md:8]: confirming the authors were added

**Review by lucidbard — COMMENTED** [review]
See comments

**Pull request merged**